zerver/lib/bugdown/__init__.py

```diff
@@ -754,11 +754,14 @@ def handleMatch(self, match):
 def make_emoji(codepoint, display_string):
     # type: (Text, Text) -> Element
     src = '/static/generated/emoji/images/emoji/unicode/%s.png' % (codepoint,)
+    # Replace underscore in emoji's title with space
+    title = display_string[1:-1].replace("_", " ")
+
     elt = markdown.util.etree.Element('img')
     elt.set('src', src)
     elt.set('class', 'emoji')
     elt.set("alt", display_string)
```

Comment:
Hmm.  I guess the challenge is that for sighted users, a very common thing to want to do is see which emoji something was so you can make it yourself, and using title to support mousing is a natural path to do that.  

Maybe we should move this discussion to a new issue about the correct accessibility behavior here (and maybe test some tools against Twitter's syntax?).